testing/btest/scripts/policy/frameworks/telemetry/prometheus.zeek

```diff
@@ -7,82 +7,100 @@
 # @TEST-PORT: BROKER_PORT2
 # @TEST-PORT: BROKER_PORT3
 # @TEST-PORT: BROKER_PORT4
-# @TEST-PORT: BROKER_PORT4
-# @TEST-PORT: BROKER_TEST_METRICS_PORT
+# @TEST-PORT: METRICS_PORT1
```

Comment:
Out of curiosity Tim, what was the problem with the additional `@TEST-PORT` uses you encountered before? Looks like it works as intended now?

Reply:
I have no idea. I think there was some conflict with the ports it was using, and I thought it was the `@TEST-PORT`, since when I took it out it worked. I've since fixed whatever was wrong with it, so now this is working.